packages/test/test-end-to-end-tests/src/test/summarizeRestart.spec.ts

```diff
@@ -57,7 +57,7 @@ describeNoCompat("Summarizer closes instead of refreshing", (getTestObjectProvid
 			{
 				eventName: "fluid:telemetry:Summarizer:Running:Summarize_cancel",
 				category: "generic",
-				error: "disconnected",
```

Comment:
This is interesting. So previously the RunningSummarizer wouldn't get any signal but the disconnect?

Reply:
Exactly